tests/perf-test/BaseOptionsList.perf-test.tsx

```diff
@@ -11,6 +11,11 @@ type BaseOptionsListWrapperProps = {
     canSelectMultipleOptions?: boolean;
 };
 
+jest.mock('@src/libs/actions/Timing', () => ({
```

Comment:
can't we globalize this mock?

Reply:
good point, thanks! I moved the mock to the global scope